src/library/scala/Proxy.scala

```diff
@@ -26,6 +26,7 @@ package scala
  *  @author  Matthias Zenger
  *  @since   1.0
```

Comment:
I was curious, so here are all the known remnants of Scala 1.0:
```shell
% rg -c "@since *1.0" src
src/library/scala/Function.scala:1
src/library/scala/PartialFunction.scala:1
src/library/scala/Proxy.scala:1
src/library/scala/Option.scala:2
src/library/scala/Console.scala:1
src/library/scala/Array.scala:2
src/library/scala/collection/immutable/List.scala:1
```

Reply:
An appropriate list. Just `Console` sticks out.